components/pipeline-service/base/external-secrets/pipelines-as-code/kustomization.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - pipelines-as-code-secret.yaml
-namespace: pipelines-as-code
+namespace: openshift-pipelines
```

Comment:
this is changing production as well.

Reply:
right re https://github.com/redhat-appstudio/infra-deployments/pull/2094#discussion_r1259923209 ... discussing in slack